aioshelly/wsrpc.py

```diff
@@ -38,15 +42,23 @@ class RPCCall:
     """RPCCall class."""
 
     def __init__(
-        self, call_id: int, method: str, params: dict[str, Any] | None, route: RouteData
+        self,
+        call_id: int,
+        method: str,
+        params: dict[str, Any] | None,
+        route: RouteData,
     ):
         """Initialize RPC class."""
         self.call_id = call_id
-        self.params = params
         self.method = method
+        self.params = params
         self.src = route.src
         self.dst = route.dst
         self.resolve: asyncio.Future = asyncio.Future()
+        self.auth = {}
+        if params and "auth" in params:
```

Comment:
not sure this is the best way to accomplish handling auth... mostly trying to avoid pylint's "too-many-arguments"

Reply:
We should not do workarounds to bypass pylint, we can either increase this limit in pylint or add it to the route class.
If it is added to the route class the class will be needed to be renamed to a better name, such as `ConnectionData`